week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -24,22 +22,21 @@ function greaterThan10() {
 
 function sortArray() {
   let letters = ["a", "n", "c", "e", "z", "f"];
-  let sortedLetters;
-
+  letters.sort();
+  let sortedLetters = letters;
```

Comment:
This could be shortened to 

`let sortedLetters = letters.sort()`

You dont need to re-assign the letters array to the sortedLetters